.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: python
+python:
+- '2.7'
+- '3.4'
+- '3.5'
+install:
+- pip install pymodbus pyserial || pip install pymodbus3 pyserial
```

Comment:
this is no good, as I commented on the other PR, there should be `install: "pip install -r requirements.txt"` here. And there is a way to set conditions in `requirements.txt` - look at this:
https://www.python.org/dev/peps/pep-0496/#id6